2025-02-17 02:38:37,557 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:38:54,465 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:12,229 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:32,147 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:39:56,043 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:40:32,719 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:40:49,641 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:41:07,408 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:41:27,172 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:41:50,919 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:42:27,579 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:42:44,336 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:43:02,071 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:43:33,743 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:43:50,639 - WARNING - Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:44:08,543 - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:44:28,316 - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:44:52,068 - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
2025-02-17 02:45:28,748 - WARNING - Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='quests-usage-dev.prod.zettablock.com', port=443): Read timed out. (read timeout=15)")': /api/user/0x6646de28934127ba20ea5444206cfd1c382a7c17/stats
